### PR TITLE
8.1 what's new and upgrade guide

### PR DESCRIPTION
**Pull request by Pessimistress**
For #4317

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -2,6 +2,109 @@
...
+
+New props are added to better control the layer's behavior:
+
+- `maxCacheByteSize`: for precise management of memory usage
+- `refinementStrategy`: to reduce flashing/overlapping during loading
+- `tileSize` (non-geospatial only)
+
+#### MVTTileLayer
+
+Based on the `TileLayer`, [MVTTileLayer](/docs/layers/mvt-layer.md) loads and renders tiles in the [Mapbox Vector Tiles specification](https://github.com/mapbox/vector-tile-spec). This new layer make it easier to leverage the many great [open source tools](https://github.com/mapbox/awesome-vector-tiles) in use with deck.gl.
```
Comment (Pessimistress) [inline docs/whats-new.md]: @jesusbotella

File: docs/whats-new.md [hunk 2]
```diff
@@ -2,6 +2,109 @@
...
+
+This effort is led by contributors from [CARTO](https://carto.com/).
+
+#### TerrainLayer
+
+[TerrainLayer](/docs/layers/terrain-layer.md) loads color-encoded heightmap and reconstructs 3D mesh surfaces. Check out our [example](https://github.com/uber/deck.gl/tree/master/examples/website/terrain).
```
Comment (Pessimistress) [inline docs/whats-new.md]: @chrisgervang

File: docs/whats-new.md [hunk 3]
```diff
@@ -2,6 +2,109 @@
...
+
+### ESRI + deck.gl
```
Comment (Pessimistress) [inline docs/whats-new.md]: @damix911

File: docs/whats-new.md [hunk 4]
```diff
@@ -2,6 +2,109 @@
...
+
+![Repeating worlds at low zoom levels](https://user-images.githubusercontent.com/2059298/71849768-b397a700-3087-11ea-8c48-93d8bb9188db.gif)
+
+The `MapView` now supports repeating worlds at low zoom levels. For backward compatibility, this feature is opt-in. Apps may turn it on by setting `views: new MapView({repeat: true})` on `Deck` or `DeckGL`.
+
+Repeating is always on when using [MapboxLayer](/docs/api-reference/mapbox/mapbox-layer.md) and [GoogleMapsOverlay](/docs/api-reference/google-maps/google-maps-overlay.md).
+
+As a result, `GoogleMapsOverlay` now supports all Google Maps zoom levels.
+
+### pydeck 0.3.0
```
Comment (Pessimistress) [inline docs/whats-new.md]: @ajduberstein

File: docs/whats-new.md (hunk 3 above)
Comment (damix911) [reply docs/whats-new.md]: Perfect! I wonder if the word _"experimental"_ should be stressed in the API docs too.

I opened a PR for it: #4396. It also fixes a couple of links in the ArcGIS API docs.

**Pull request merged**